iosApp/iosApp/ContentView.swift

```diff
@@ -107,28 +92,24 @@ struct ContentView: View {
             OnboardingPage(screens: onboardingScreensPending, onFinish: {
                 contentVM.onboardingScreensPending = []
             })
-        } else if selectedTab == .more {
+        } else {
             TabView(selection: $selectedTab) {
                 nearbyTab
                     .tag(SelectedTab.nearby)
-                    .tabItem { TabLabel(tabText(.nearby), image: .tabIconNearby) }
+                    .tabItem { TabLabel(tab: SelectedTab.nearby) }
                 MorePage(viewModel: settingsVM)
                     .tag(SelectedTab.more)
-                    .tabItem { TabLabel(tabText(.more), image: .tabIconMore) }
+                    .tabItem { TabLabel(tab: SelectedTab.more) }
                     .onAppear { analytics.track(screen: .settings) }
             }
-        } else {
-            nearbyTab
         }
     }
 
     @State var selectedDetent: PresentationDetent = .halfScreen
     @State var visibleNearbySheet: SheetNavigationStackEntry = .nearby
     @State private var showingLocationPermissionAlert = false
 
-    @ViewBuilder var nearbySheetContents: some View {
-        // Putting the TabView in a VStack prevents the tabs from covering the nearby transit contents
-        // when re-opening nearby transit
```

Comment:
suggestion (non-blocking): I think it would be worth still keeping this comment & the ViewBuilder annotations